app/src/Components/Event/EventSummary.tsx

```diff
@@ -27,7 +28,11 @@ const RightColumn = styled.View`
 `
 
 const EventSummary: FC<EventSummaryProps> = ({ data }) => (
-  <EventDetails key={data.id}>
+  <EventDetails
+    key={data.id}
+    onPress={() => {
+      console.log('Event detail clicked');
```

Comment:
Do we need to keep this in or was it just for testing?

Reply:
Testing - taken it out now.